tests/_tests_scripts/dl_z_mvp_mnist_metric_learning.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+
+from catalyst import data, dl
+from catalyst.contrib import datasets, models, nn
+import catalyst.contrib.data.transforms as t
+from catalyst.utils import get_device
```

Comment:
so, looks like you can
```
from catalyst import data, dl, utils

utils.get_device()
```

it also looks like we need to simplify contrib for more flatten usage :)

Reply:
done